src/components/simplified-exchange-component.ts

```
@@ -1026,14 +1316,20 @@ export class SimplifiedExchangeComponent {
           break;
 
         case "swap":
-          const fromToken = this._state.direction === "deposit" ? ("LUSD" as const) : ("UUSD" as const);
-          const toToken = this._state.direction === "deposit" ? ("UUSD" as const) : ("LUSD" as const);
+          const fromToken = this._state.direction === "deposit"
+            ? ("LUSD" as const)
+            : ("UUSD" as const);
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)